ush/parsing_model_configure_FV3.sh

```diff
@@ -19,29 +19,20 @@ fi
 
 rm -f model_configure
 cat >> model_configure <<EOF
-total_member:            $ENS_NUM
```

Comment:
Is removing this going to be an issue for ensembles (either EnKF or GEFS)? I know this branch is now a dead-end, but we still want to be able to pull changes over to the sprint branch, and member number is a field in the output file.

Reply:
This is a variable that was only used in the NEMS driver and it wasn't even really being used from what I remember, so I don't know how that would effect the Grib headers, but finding where this was removed in NEMS could perhaps give you more answers.